### PR TITLE
Audit fixes: part 1

### PR DESCRIPTION
**Pull request by mrjbq7**
Breaking #39061 into smaller pieces.

First part has fixes for using Ruby commands and changing the freedesktop.org URLs.

### CODE REVIEW

**Comment by mrjbq7** [comment]
(I just realized this is 16 files changed, I hope thats okay.)

File: Library/Formula/cmigemo.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ class Cmigemo < Formula
   depends_on 'nkf' => :build
 
   def install
-    system "chmod +x ./configure"
+    chmod "+x", "./configure"
```
Comment (DomT4) [inline Library/Formula/cmigemo.rb]: ` 0755 ` rather than ` +x `.

**Comment by mrjbq7** [comment]
Fixed chmod +x.

**Comment by mrjbq7** [comment]
Rebased on current master.

**Comment by mikemcquaid** [comment]
Similarly let's do `brew audit --strict` here. `brew style --fix` will autofix issues it can. Also, let's hold off creating more PRs for these until each one is merged.

**Comment by mrjbq7** [comment]
I'd love to know how to help you guys merge these patches.  I don't mind doing the extra work of splitting into small patches, and I don't mind if it takes days or weeks to get them done, but I'm kinda just waiting here until you're ready to proceed?

**Comment by DomT4** [comment]
Mike can punch in any corrections if I'm wrong, but:

* Fix everything flagged by ` brew audit --strict `.
* No more than 10 formulae per PR. (Per Jack)
* Make sure the test bot is passing.
* Wait for one PR to be merged before creating another.

**Comment by mrjbq7** [comment]
Wait, so rather than fix issues by "type" (like in this request), I should fix all issues by formula reported by ``--strict``?

**Comment by mrjbq7** [comment]
I figured I would fix errors by group (e.g., freedesktop.org URLs, then apache.org, then github.io, etc.).  It's super easy to do search and replace and then pull request, and usually affects different formulas.

**Comment by mrjbq7** [comment]
Also, the test bot seems to fail for reasons unrelated to this pull request.  Not sure what to do about that.

**Comment by DomT4** [comment]
> Wait, so rather than fix issues by "type" (like in this request), 

It doesn't really matter how you sort them, whether by type or random order. It's just preferred the the changes made are more thorough than just one fix.

You can fix them by type if it's easier for you, just fix everything flagged by ` brew audit --strict ` rather than the issues flagged by ` brew audit `. (` audit --strict ` will contain everything in ` brew audit `).

> Also, the test bot seems to fail for reasons unrelated to this pull request. Not sure what to do about that.

If it's a build error, it needs fixing.

**Comment by mrjbq7** [comment]
Okay, then I think we should start merging these first 8 pull requests (#39069 is the first and #39133 is the second).

After we get these done, I'll start working on the additional strict failures.

**Comment by mikemcquaid** [comment]
Also: one commit per formula, the build needs to be green (if things are broken we need to try and fix them), if you're fixing `audits` you should fix all the `audit --strict` warnings too.

**Comment by mrjbq7** [comment]
Okay, then closing these PRs. Will change approach and open another later.

**Pull request closed without merging**